src/client/src/region.rs

```diff
@@ -0,0 +1,183 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use api::v1::region::region_request::Body as PbRegionRequest;
+use api::v1::region::{self as region_pb};
+use api::v1::{AuthHeader, RequestHeader as PbRequestHeader, ResponseHeader as PbResponseHeader};
```

Comment:
Now no structs have naming conflicts. We can import `region_request` and use `region_request::Body`.
```suggestion
use api::v1::region::region_request{self, RegionRequest};
use api::v1::{AuthHeader, RequestHeader, ResponseHeader};
```

Examples:
https://github.com/GreptimeTeam/greptimedb/pull/2269/files#diff-190ca5fa9f563aa180bb7ca434d4e3d0c3bc5d9991d8d83cc7a686218e60f157R337-R347

Reply:
Alias is confusing and sometimes it is hard for me to search them, distinguish them, and map them to the correct pb defines.

e.g. `PbRegionRequest`, `region_pb::RegionRequest`.......